x-pack/plugins/ingest_manager/common/mocks.ts

```diff
@@ -44,3 +44,90 @@ export const createPackageConfigMock = (): PackageConfig => {
     ],
   };
 };
+
+export const createPackageConfigWithInitialManifestMock = (): PackageConfig => {
```

Comment:
maybe me more specific in the naming, any reason to no keep this in the security_solution plugin? it will make future change easier no?   

```suggestion
export const createEndpointPackageConfigWithInitialManifestMock = (): PackageConfig => {
```

Reply:
@nchaulet These are mocks related to the policy, so IMO they belong here. The manifest is part of the policy. I'm happy to rename or remove though. Can we address in a quick follow-up to this one?

Reply:
We discussed and in our next PR, I'll see if I can make the names more descriptive. Leaving them here for now, as they may be useful for future tests. We can move them to security_solution if they don't end up being useful here.

Reply:
@madirey sorry for the late response.
I agree with @nchaulet on this here - these are endpoint specific mocks that Ingest knows nothing about. We only use their data structure to store our Policy data in (the area where we store it accepts any json blob).

We should really move them to our Plugin (probably under the generator we have in `/common/endpoint`.